### PR TITLE
SALTO-6477: Support new IssueView format

### PR DESCRIPTION
**Pull request by idoamit8**
In this PR I supported new issueView format, only for stage 1.

---

_Additional context for reviewer_
Much much more context can be found in [SALTO-6477 ](https://salto-io.atlassian.net/browse/SALTO-6477?focusedCommentId=174771)

---
_Release Notes_: 
_Jira Adapter:_
* [SALTO-6477 ](https://salto-io.atlassian.net/browse/SALTO-6477?focusedCommentId=174771): Supporting new issue view format stage 1. 

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/jira-adapter/src/filters/layouts/layout_types.ts [hunk 1]
```diff
@@ -88,6 +88,12 @@ export type ContainerIssueLayoutResponse = {
     nodes: {
       fieldItemId?: string
       panelItemId?: string
+      items?: {
+        nodes: {
```
Comment (IdoZakk) [inline packages/jira-adapter/src/filters/layouts/layout_types.ts:92]: There is a small mismatch here with the scheme- if nodes is required add to the scheme, if not add a ? here.

**Review by IdoZakk — APPROVED** [review]
Nice!
See my comments
Do we need noise reduction?

File: packages/jira-adapter/src/filters/layouts/layout_service_operations.ts [hunk 2]
```diff
@@ -148,8 +148,23 @@ const fromLayoutConfigRespToLayoutConfig = (layoutConfig: IssueLayoutConfigurati
       }),
     )
     .filter(isLayoutConfigItem)
-
-  return { items }
+  const defaultContentItems = containers
+    .flatMap(container =>
+      container.items.nodes?.[0]?.items?.nodes?.map(innerNode => {
```
[the remark this answers is not in the document]
Comment (idoamit8) [reply packages/jira-adapter/src/filters/layouts/layout_service_operations.ts:153]: @IdoZakk Indeed nodes is a required field. I remove the `?` there.
the second item field is not a required field.

File: packages/jira-adapter/src/filters/layouts/layout_service_operations.ts [hunk 3]
```diff
@@ -148,8 +148,23 @@ const fromLayoutConfigRespToLayoutConfig = (layoutConfig: IssueLayoutConfigurati
...
+        const dataKey = innerNode.panelItemId ?? innerNode.fieldItemId
+        if (dataKey === undefined) {
+          return undefined
+        }
+        return {
+          type: innerNode.fieldItemId ? 'FIELD' : 'PANEL',
```
[the remark this answers is not in the document]
Comment (idoamit8) [reply packages/jira-adapter/src/filters/layouts/layout_service_operations.ts:159]: a node can not be both panel and field, but I changed it for readability

File: packages/jira-adapter/src/filters/layouts/layout_types.ts (hunk 1 above)
Comment (idoamit8) [reply packages/jira-adapter/src/filters/layouts/layout_types.ts:92]: It there are no items. (meaning are no nodes as well) - the schema guard will pass?

**Pull request merged**